web/src/message_edit.js

```diff
@@ -484,27 +489,22 @@ function edit_message($row, raw_content) {
         .find(".message-edit-feature-group .audio_link")
         .toggle(compose_call.compute_show_audio_chat_button());
 
-    const $message_edit_content = $row.find("textarea.message_edit_content");
     const $message_edit_countdown_timer = $row.find(".message_edit_countdown_timer");
     const $copy_message = $row.find(".copy_message");
 
     if (!is_editable) {
         $message_edit_content.attr("readonly", "readonly");
-        create_copy_to_clipboard_handler($row, $copy_message[0], message.id);
+        create_copy_to_clipboard_handler($row, $copy_message[0], $message_edit_content);
     } else {
         $copy_message.remove();
-        const edit_id = `#edit_form_${CSS.escape(rows.id($row))} .message_edit_content`;
-        const listeners = resize.watch_manual_resize(edit_id);
-        if (listeners) {
-            currently_editing_messages.get(rows.id($row)).listeners = listeners;
-        }
-        composebox_typeahead.initialize_compose_typeahead(edit_id);
-        compose_ui.handle_keyup(null, $(edit_id).expectOne());
-        $(edit_id).on("keydown", function (event) {
-            compose_ui.handle_keydown(event, $(this).expectOne());
+        resize.watch_manual_resize_for_element($message_edit_content[0]);
+        composebox_typeahead.initialize_compose_typeahead($message_edit_content[0]);
```

Comment:
This function expects a selector, not an element, so I expect this to break typeahead.

Reply:
I guess it doesn't, because this works:
```
export function initialize_compose_typeahead(selector) {                                                
    const bootstrap_typeahead_input = {                                                                 
        $element: $(selector),                                                                          
        type: "textarea",                                                                               
    };                                                                                                  
```

Probably we should change that API but given there's a current TS migration PR for that file, I'll defer it. @evykassirer FYI.

Reply:
#30120